historical_system_profiles/probes.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+from historical_system_profiles import config
+
+
+def _update_liveness_state():
+    # touches liveness probe file
+    Path(config.liveness_probe_filepath).touch(exist_ok=True)
```

Comment:
What would happen in an env that does not have `liveness_probe_filepath` set by an env var, like in a dev setup?

Reply:
Values of `liveness_probe_filepath` and `readiness_probe_filepath` have defaults set in https://github.com/RedHatInsights/historical-system-profiles-backend/pull/234/files#diff-f8f1173880fcb4a20eaa4751d764b459cf8b8e3e6c9dfe57bd01f50a00cd3962R127